app/models/accounting/quickbooks/data_extractor.rb

```diff
@@ -1,13 +1,15 @@
 module Accounting
   module Quickbooks
     class DataExtractor
-      attr_reader :txn
+      attr_reader :txn, :loan
 
       def initialize(txn)
         @txn = txn
+        @loan = txn.project
       end
 
       def extract!
+        # TODO: if 'other', managed: false
```

Comment:
@meltheadorable, this is breaking at the moment because it requires the txn to have an account in the transaction model